profile-service/src/test/java/com/telecominfraproject/wlan/profile/controller/ProfileControllerTest.java

```diff
@@ -76,6 +75,25 @@ public void testProfileCRUD() throws Exception {
         profileController.delete(ret.getId());
         
     }
+    
```

Comment:
I would move this test into ProfileServiceRemoteTest, and also add a test for the update() operation to make sure that name uniqueness is also enforced there